<?xml version="1.0"?>
<launch>
  <arg name="uri3" default="radio://0/35/2M/E7E7E7E703" />
  <arg name="frame3" default="/vicon/crazyflie3/crazyflie3" />

  <arg name="uri4" default="radio://0/80/2M/E7E7E7E704" />
  <arg name="frame4" default="/vicon/crazyflie4/crazyflie4" />

  <arg name="uri5" default="radio://0/57/2M/E7E7E7E705" />
  <arg name="frame5" default="/vicon/crazyflie5/crazyflie5" />

  <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch">
  </include>

  <!-- <group ns="crazyflie3">
    <include file="$(find crazyflie_driver)/launch/crazyflie_add.launch">
      <arg name="uri" value="$(arg uri3)" />
      <arg name="tf_prefix" value="crazyflie3" />
      <arg name="enable_logging" value="False" />
      <arg name="roll_trim" value="0.1" />
      <arg name="pitch_trim" value="0.1" />
    </include>

    <node name="controller" pkg="crazyflie_demo" type="a_standing_wave_cf3.py" output="screen"> 
      <param name="start_time" command="date +'%d-%m-%Y_%Ih%Mm%S.%N'"/>
    </node>
  </group> -->

  <group ns="crazyflie4">
    <include file="$(find crazyflie_driver)/launch/crazyflie_add.launch">
      <arg name="uri" value="$(arg uri4)" />
      <arg name="tf_prefix" value="crazyflie4" />
      <arg name="enable_logging" value="False" />
      <arg name="roll_trim" value="-0.6" />
      <arg name="pitch_trim" value="0.1" />
    </include>

    <node name="controller" pkg="crazyflie_demo" type="a_standing_wave_cf4.py" output="screen"> 
      <param name="start_time" command="date +'%d-%m-%Y_%Ih%Mm%S.%N'"/>
    </node>
  </group>

  <!-- <group ns="crazyflie5">
    <include file="$(find crazyflie_driver)/launch/crazyflie_add.launch">
      <arg name="uri" value="$(arg uri5)" />
      <arg name="tf_prefix" value="crazyflie5" />
      <arg name="enable_logging" value="False" />
      <arg name="roll_trim" value="0.5" />
      <arg name="pitch_trim" value="-0.8" />
    </include>

    <node name="controller" pkg="crazyflie_demo" type="a_standing_wave_cf5.py" output="screen"> 
      <param name="start_time" command="date +'%d-%m-%Y_%Ih%Mm%S.%N'"/>
    </node>
  </group> -->

  <include file="$(find vicon_bridge)/launch/vicon.launch"/>
</launch>